Cargo.toml

```diff
@@ -55,3 +55,4 @@ std = ["curve25519-dalek/std"]
 nightly = ["curve25519-dalek/nightly"]
 u64_backend = ["curve25519-dalek/u64_backend"]
 u32_backend = ["curve25519-dalek/u32_backend"]
+
```

Comment:
Nit: empty line

Reply:
The trailing newline is getting added by CLion, and is actually required by rustfmt, so I'm not sure I want to fight it too strongly.